pyscf/pbc/grad/krks.py

```diff
@@ -81,6 +81,8 @@ def get_vxc(ni, cell, grids, xc_code, dms, kpts, kpts_band=None, relativity=0, h
             for i in range(nset):
                 rho = make_rho(i, ao_k2[:,0], mask, xctype)
                 vxc = ni.eval_xc(xc_code, rho, 0, relativity, 1)[1]
+                if vxc is None:
```

Comment:
why can vxc be None? Is it from xctype=None or HF? should this special vxc be handled in a separated case?

Reply:
This happens when `mf.xc=""`, which corresponds to a J only calculation.